broker/lib/middleware.js

```diff
@@ -75,7 +75,10 @@ exports.checkQuota = function () {
       next();
     } else {
       const platform = _.get(req, 'body.context.platform');
-      if (platform === CONST.PLATFORM.CF) {
+      const origin = _.get(req, 'body.context.origin');
+      if (platform === CONST.PLATFORM.CF ||
```

Comment:
can the evaluation in this `if` clause be encapsulated in a small method named `shouldCheckQuotaForPlatform()` for better readability? 

Also, all APIs except delete has context so can you keep this to:
```
const orgId = req.body.context.organization_guid;
```

Reply:
Done